domain-cc/cc-app/src/python_src/pydantic_models.py

```diff
@@ -10,7 +10,7 @@ class Claim(BaseModel):
     claim_type: str = "claim_for_increase"
     contention_text: Optional[str]  # marked optional to retain compatibility with v1
 
-    @root_validator()
+    @root_validator(pre="before")
```

Comment:
Should bump pydantic + fastapi versions, switch to `@model_validator` then, added a ticket to tackle this tech debt [here](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/374)